source/internals/options.js

```diff
@@ -70,6 +94,10 @@ const askiOSPods = () =>
       wipeiOSBuild = true;
       return resolve();
     }
+    if (args.includes('--keep-iOS-pods')) {
```

Comment:
Isn't this mutually exclusive with the `Wipe iOS Pods folder? (Y/n)`? If the user says no, then it automatically keeps the pods.

The same for the remaining options.